### DISCUSSION TITLE
Default components height

### DISCUSSION THREAD

**Discussion by bucari** [saadeghi/daisyui#2344] [Q&A]
Hi,
i added in app.css 

```
.input {
    @apply h-10;
}
.select {
    @apply h-10;
}
```
to customize component height, but it works only for input component. How can i change the default height for a select or better for all components?

**Comment by saadeghi** [comment]
All components don't have the same height.  
...So It's not possible to change them all at once. You can apply customizations to each component.

Another way is changing the font-size of root element. All daisyUI sizes are `rem` if you want absolutely everything to get smaller or lager, you can do that: https://play.tailwindcss.com/SglxgnHtL9?file=css

**Comment by bucari** [comment]
so how can i make select compoenent the same height of input text, like this

![image](https://github.com/saadeghi/daisyui/assets/97893337/b6ccaf90-e9d4-46d6-ae82-d28654f8e090)

 this works for input text component
```
.input {
    @apply h-10;  
}
```
 but this 

```
.select{
    @apply h-10;
}

```
doesn't for select component

**Comment by bucari** [comment]
i found the solution

```
.select {
    @apply h-10;
    min-height:initial;
}
```
